scripts/signTransaction.js

```diff
@@ -0,0 +1,125 @@
+require('ts-node/register');
```

Comment:
@nadezhdapopovaa What is the purpose of this file?

Reply:
you are correct - it's not supposed to be there. will fix it